generators/app/templates/src/app/components/FormInput/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+interface Props {
+  className?: string;
+  disabled?: boolean;
+  error?: string | undefined;
+  errorClassName?: string;
+  inputClassName?: string;
+  isTextarea?: boolean;
+  inputType: string;
+  label?: string | object | undefined;
+  labelClassName?: string;
+  name: string;
+  onBlur?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  onChange: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  onFocus?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  placeholder?: string;
+  readOnly?: boolean;
```

Comment:
Same as before, check if this props are optional or not and remove de default value for the optional ones or the marker for the mandatory ones.

Reply:
As I mentioned in another PR, I don't think this is the case with TS, but I'm waiting pablo's confirmation

Reply:
@damfinkel  @pabloferro, Could you talk about this?